ushahidi/platform-client
# Remove Form Caching

This pull request makes the following changes:
-removed all caching and associate code from anything relating to forms.

Testing checklist:
- [ ] Everything should work exactly how it did before
- [ ] Local storage should not have anything in it related to forms.

- [ ] Surveys
  - Open Settings->Surveys, in a separate Tab open Settings->Surveys again
    - [ ] In first tab, add a new survey
    - [ ] Refresh second tab and ensure new Survey appears
  - Open Map view, in a separate Tab open Settings->Surveys
    - [ ] On Settings->Surveys, add a new Survey
    - [ ] Refresh Map 
      - [ ] Ensure new Survey appears in Filter menu
      - [ ] Ensure new Survey appears in Mode Context menu
  - Open Map view, in a separate Tab open Settings->Surveys
    - [ ] On Settings->Surveys, add a new Survey
    - [ ] Refresh Map 
      - [ ] Ensure new Survey appears in Filter menu
- [ ] Posts
  - Open Settings->Surveys, in a separate Tab open Map View
    - [ ] Add a new Survey
    - [ ] In Map View, refresh, ensure that the add menu contains the new Survey
  - Open Settings->Surveys, in a separate Tab open Data View
    - [ ] Add a new Survey
    - [ ] In Data View, refresh, ensure that the add menu contains the new Survey
  - Open Settings->Surveys, in a separate Tab open Map View
    - [ ] Add a new Survey
    - [ ] In Map View, refresh, ensure that the add menu contains the new Survey
  - Open Settings->Surveys, in a separate Tab open Map View
    - [ ] Add a new Survey
    - [ ] In Map View, refresh, ensure that the add menu contains the new Survey
    - [ ] Add a Post of this Survey type
    - [ ] In Settings->Surveys, add a new field to the Survey
    - [ ] Edit the previously create Post, ensure new field appears
    - [ ] In Settings->Surveys, add a new task to the Survey
    - [ ] Edit the previously create Post, ensure new task appears    
  - Open Settings->Surveys, in a separate Tab open Data View
    - [ ] Add a new Survey
    - [ ] In Data View, refresh, ensure that the add menu contains the new Survey
    - [ ] Add a Post of this Survey type
    - [ ] In Settings->Surveys, add a new field to the Survey
    - [ ] Edit the previously create Post, ensure new field appears
    - [ ] In Settings->Surveys, add a new task to the Survey
    - [ ] Edit the previously create Post, ensure new task appears

Fixes ushahidi/platform# .

Ping @ushahidi/platform
